include/morphio/endoplasmic_reticulum.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <morphio/types.h>
+
+namespace morphio {
+/**
+ * The entry-point class to access endoplasmic reticulum data
+ *
+ * Spec https://bbpteam.epfl.ch/documentation/projects/Morphology%20Documentation/latest/h5v1.html
+ **/
+class EndoplasmicReticulum
+{
+public:
+    /**
+       Returns the list of neuronal section indices
+    **/
+    const std::vector<uint32_t>& sectionIndices() const;
+
+    /**
+       Returns the volumes for each neuronal section
+    **/
+    const std::vector<float>& volumes() const;
+
+    /**
+       Returns the surface areas for each neuronal section
+    **/
+    const std::vector<float>& surfaceAreas() const;
+
+    /**
+       Returns the number of filaments for each neuronal section
+    **/
+    const std::vector<uint32_t>& filamentCounts() const;
+
+private:
+    EndoplasmicReticulum(std::shared_ptr<Property::Properties> properties)
+        : _properties(properties)
```

Comment:
To save an atomic increment in `_ptr` constructor and an atomic decrement in `properties` destructor:

```suggestion
        : _properties(std::move(properties))
```

Reply:
Good point. Thanks for the suggestion.